package.json

```diff
@@ -5,13 +5,19 @@
   "description": "Foreman isn't really a node module, these are just dependencies needed to build the webpack bundle. 'dependencies' are the asset libraries in use and 'devDependencies' are used for the build process.",
   "private": true,
   "devDependencies": {
+    "babel-cli": "^6.10.1",
     "babel-core": "~6.7.2",
     "babel-eslint": "^6.1.2",
     "babel-loader": "~6.2.4",
+    "babel-plugin-transform-class-properties": "^6.11.5",
```

Comment:
This is currently a stage 2 proposal (i.e. not part of es2016 even, no telling if and when it will be standard). It is only used once and that instance could be handled just as well without using non-standard features.